packages/dashboard/src/vaadin-dashboard-section.js

```diff
@@ -203,6 +210,32 @@ class DashboardSection extends DashboardItemMixin(ElementMixin(ThemableMixin(Pol
       this.setAttribute('role', 'section');
     }
   }
+
+  connectedCallback() {
+    super.connectedCallback();
+    const dashboard = findAncestorInstance(this, Dashboard);
+    if (dashboard) {
+      this.__rootHeadingLevel = dashboard.rootHeadingLevel;
+      this.__rootHeadingLevelChangedHandler = (e) => {
+        if (this.__rootHeadingLevel !== e.detail.value) {
+          this.__rootHeadingLevel = e.detail.value;
+        }
+      };
+      dashboard.addEventListener('root-heading-level-changed', this.__rootHeadingLevelChangedHandler);
```

Comment:
Instead of introducing another data synchronization mechanism (event listeners on the root, and findAncestorInstance...), we could synchronize root heading level the same way as `i18n` is currently synchronized.

Reply:
Makes sense. Removed the event mechanism and used the i18n way.